azurerm/internal/services/managedservices/data_source_lighthouse_definition.go

```diff
@@ -0,0 +1,107 @@
+package managedservices
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmLighthouseDefinition() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmLighthouseDefinitionRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"registration_definition_id": {
```

Comment:
@ArcturusZhang @katbyte - Do you think that registration_definition_id and registration_definition_name should be renamed to lighthouse_definition_id and lighthouse_definition_name to match the resource name azurerm_lighthouse_dinition?

Reply:
thats a good point! yes anytime we are referrring to another resources name property or id we should be explicit on what resource type we are referring to

Reply:
Sure. I will change this as well to lighthouse_definition_id and lighthouse_definition_name